src/qiskit_sphinx_theme/assets/styles/_left-sidebar.scss

```diff
@@ -127,15 +127,15 @@ body {
   width: 100%;
 }
 
-/* Based off of `.sidebar-tree .reference:hover`. */
+// Based off of `.sidebar-tree .reference:hover`. 
 .qiskit-translations-list-container a:hover,
 .qiskit-previous-releases-list-container a:hover {
   background: var(--color-sidebar-item-background--hover);
 }
 
-/* --------------------------------------------------------------------------------------
-* Translations
-* -------------------------------------------------------------------------------------- */
+// --------------------------------------------------------------------------------------
+// Translations
+// --------------------------------------------------------------------------------------
```

Comment:
I reordered code in the below two sections. No changes made, other than to use `//` comments.